src/zope/container/btree.py

```diff
@@ -25,8 +25,8 @@
 
 
 class Lazy(object):
-    """Lazy Attributes.
-    """
+    #Lazy Attributes.
+
```

Comment:
I don't believe `Lazy` is an intended API export from this package, and so didn't want to see it documented. If users want a Lazy attribute, that's what `zope.cachedescriptors` is for. But I also didn't want to break anyone that might be using it, so renaming it to `_Lazy` was out, as was creating an `__all__` attribute that left it out (in case someone was using a * import).